File: DotNetCoreBackend/Models/Room.cs
namespace DotNetCoreBackend.Models
{
    public enum RoomStatus {
        Idle = 0,
        Busy = 1
    }


    public class Room : Base
    {
        public string Name { get; set; }

        public int floor { get; set; }

        public RoomStatus status { get; set; }

        public long LastUpdateAt { get; set; }
    }
}

File: JavaBackground/src/main/java/com/maple/model/Room.java
package com.maple.model;

import javax.persistence.Entity;
import javax.persistence.EnumType;
import javax.persistence.Enumerated;

@Entity
public class Room extends Base {
    private String name;
    
    private int floor;
    
    @Enumerated(EnumType.ORDINAL)
    private RoomStatus status;
    
    private long lastUpdateAt;
}
